apis/apiextensions/v1/composition_types.go

```diff
@@ -277,6 +346,8 @@ func (c *Patch) Apply(from, to runtime.Object, only ...PatchType) error {
 		return c.applyFromFieldPathPatch(from, to)
 	case PatchTypeToCompositeFieldPath:
 		return c.applyFromFieldPathPatch(to, from)
+	case PatchTypeFromConstantValue:
+		return c.applyFromConstantValuePatch(to)
```

Comment:
Nothing to do in this particular PR, but we should get rid of this "from" and "to" notation sometime soon. It made sense originally, but less so now per https://github.com/crossplane/crossplane/pull/2137/files#r593597291.